crates/driver/src/main.rs

```diff
@@ -220,7 +220,7 @@ async fn build_submitter(common: &CommonComponents, args: &Arguments) -> Arc<Sol
         match strategy {
             TransactionStrategyArg::PublicMempool => {
                 transaction_strategies.push(TransactionStrategy::CustomNodes(StrategyArgs {
-                    submit_api: Box::new(CustomNodesApi::new(vec![web3.clone()])),
+                    submit_api: Box::new(CustomNodesApi::new(vec![web3.clone()], false)),
```

Comment:
The `PublicMempool` strategy in the new driver also uses `disable_high_risk_public_mempool_transactions` instead of hardcoding a value.

Reply:
Nice catch! This was a booboo when trying to use `false` for all `e2e` instances of `CustomNodesApi`.